atlp-rwanda/trojans-ec-bn
# #184347115 Notifications feature

### What does this PR do?

This PR reflects the functionality of the notifications, where if there is any product life cycle such as being expired, made available, removed etc happen, the appropriate user gets both email and in app notifications.

### Description of Task to be completed?

Provided a product lifecycle changes (added, removed, made-avilable, expired, bought, etc) order accepted/denied the notification should be sent to the the right user/recipient

### How should this be manually tested?

To test this manually, open the index.html file in the public directory then fill in the prompt with the right authentication token and the on the documentation try to make a product available or expired then if the toke entered in the page is of the user who have to receive the notification, you will see it.

### Any background context you want to provide?

no

### What are the relevant pivotal tracker stories?

https://www.pivotaltracker.com/story/show/184347115

### Screenshots (if appropriate)


### Questions:

# Type of change

Please check the options that are relevant.


# Checklist:
